### ISSUE TITLE
Improve front page, software list, and tool text and descriptions

### ISSUE THREAD

**Issue by ValWood** [geneontology/amigo#331]
Much better, much clearer what things are.

Would it be possible to make all options visible on a default size window on a laptop.
 I always thiank you should see the whole front page.

Currently:

![current](https://cloud.githubusercontent.com/assets/7359272/14137472/dd2e53b8-f65f-11e5-9f1d-d9f20cc8c6f9.jpg)

(Maybe the top search panel could be a bit smaller?)

Some possible  text refinements

1.
Use Search Templates to get started in exploring the Gene Ontology data.
-> Use predefined templates to explore  Gene Ontology data.

2.
Use the drill-down browser to get a basic view of the ontology structure, as well as annotation counts.
->
Use the drill-down browser to  view the ontology structure with annotation counts?

3.
Use GOOSE to query a legacy GO database with SQL or edit one of the templates.
(Does this mean
->
Use editable SQL templates  to query the GO database (why legacy?)
(Also available options here are very confusing...what are the differences for users???)

4. View the most recent statistics about the Gene Ontology data.
-> Gene Ontology annotation statistics?

5.
On the  more tools page
http://tomodachi.berkeleybop.org/amigo/software_list
There are descriptions of each tool.
Shouldn't these be the same as the text you see on the information link?

for example search templates says:

a)
Search Templates (a search "wiziard") can be used to quickly answer common questions using a fill-in-the-blank approach.
On this page,

but
Search Templates are a simple fill-in-the-blank wizard interface to AmiGO 2 users that are either unfamiliar with how to search in AmiGO 2 or for users that want to simplify common queries. To use, fill in the necessary blank columns with IDs, selecting from the autocomplete results; hit the go icon at the end of the question when done.

on the information link
(the second one is too wordy- do users really need to be told to 'hit the go icon at the end of the question when done?')

b) Statistics
Base statistics can give a high-level view of what kind of data is available in AmiGO.
or
The page show basic counts of data available in the GO dataset for AmiGO. This page require JavaScript to be enabled.

(Neither of these description is great)

c) Drill-down browser 
Drill-down browser can be used to grossly explore the ontologies from the given top levels.
and
This is a very simple top-down exploration widget for the ontologies. It does not worry about multiple paths and logic, it just lets you open up direct descendant terms one at a time.
This browser performs no lookahead, so you will not know if a term has children until you attempt to open it.
The best practice for use would be to set filters before exploration--setting/resetting filters will also reset the tree.

Neither very helpful...

**Comment by kltm** [comment]
There are two or three descriptions related to any given tool:

1. front page descriptions are just hooks to get people going in the right direction, space is very tight and the description tries to center around a single word
2. software_list descriptions should be more descriptive, as well as having some additional notes
3. tools page descriptions should be incredibly details, including instructions, quirks, links, etc.

Naturally, as I've spent way too much time with most of these things, my notes and instructions tend to be incoherent to a wider audience. As well, a lot of these text blobs are written at different times and proofed by different people (or nobody at all), which means there is some unfortunate drift in there. I'm unlikely to create a unified doc system in the near term, so we're left to go after the immediate pain points.

If you have replacement text for any of these, it's much appreciated. I've gone ahead and added most of your changes back to the landing page (keeping "legacy" in GOOSE description though).

**Comment by ValWood** [comment]
OK will continue wit fresh eyes as I test. You can choose to ignore.
What does 'legacy' mean here though? This to me implies you can only search old databases....

**Comment by ValWood** [comment]
Here you go:

a) 
Search Templates (a search "wizard") can be used to quickly answer common questions using a fill-in-the-blank approach. 


b) Statistics

 Annotation counts providing an overview of the data available in AmiGO (Page requires JavaScript).

c) Drill-down browser 

 Drill-down browser can be used to explore the ontologies and annotation from the top levels by filtering and opening direct descendant terms individually to create a custom view. 
Tip: Set your required filters before exploration, resetting filters will also reset the tree.

**Comment by kltm** [comment]
a) corrected to new text on software_list
b) the statistics are not just annotation counts, but also publications, with more items in the future; I correct to " Annotation counts and statistics that provide an overview of the GO data available in AmiGO." on the tool page and software_list, adjustment on landing page
c) partial or total text replacement in software_list and tool page
